posts/views.py

```diff
@@ -678,3 +678,15 @@ def download_csv(request, pk: int):
         headers={"Content-Disposition": f"attachment; filename={filename}.csv"},
     )
     return response
+
+
+@api_view(["GET"])
+@permission_classes([AllowAny])
+def random_post_id(request):
+    post = (
+        Post.objects.filter_permission(user=request.user)
+        .filter_active()
+        .order_by("?")
+        .first()
+    )
+    return Response({"id": post.id, "url_title": post.url_title})
```

Comment:
`url_title` is not very safe to use since it contains spaces, special chars and etc, please use `get_post_slug` instead

Reply:
Also, we have different urls for questions from communities and notebooks (see `getPostLink` function on the frontend)